_stylesheet.scss

```diff
@@ -173,6 +173,11 @@ $dock_fixed_inner_margin: $dock_side_margin;
     }
 }
 
+#dashtodockContainer.top.shrink #dash .dash-background {
```

Comment:
To be honest I think we could just stop supporting top positioning... maybe next time it prove problematic as you have already done the work to fix it.

Reply:
Yeah, I agree... We may do that indeed.

It was mostly an exercise for the rest of the logic to be working though :)